web/src/utilities/analyticsVis.ts

```diff
@@ -17,73 +18,96 @@ const defaultStructureColors = {
   gland: 'red',
 };
 
-function drawLines(target: StructurePoint) {
-  const retArray: Array<Array<Array<number>>> = [];
-  const computedLines = computed(() => store.state.nucleiToNearestGlandDistances);
+const colors = distinctColors({
+  count: 50,
```

Comment:
Since 20 glands is hard coded into the script for generating glands, would it make sense to use 20 colors instead of 50? Would that help make the colors more visually distinct from one another?